promql/promqltest/testdata/histograms.test

```diff
@@ -113,6 +113,11 @@ eval instant at 50m histogram_fraction(0, 0.2, rate(testhistogram3[10m]))
 	{start="positive"} 0.6363636363636364
 	{start="negative"} 0
 
+
+eval instant at 50m histogram_fraction(0, 0.2, rate(testhistogram3_bucket[10m]))
```

Comment:
It should be easy to add more test cases. histogram_fraction and histogram_quantile can be tested in pairs to see if the results are consistent. See native_histograms.test for inspiration.

Reply:
Added a couple more tests - also some to test interpolation. This uncovered a bug in the interpolation :see_no_evil: